lesson-11/main.js

```diff
@@ -0,0 +1,453 @@
+btnCounterAdultMinus.addEventListener('click', minusAdult);
+function minusAdult() {
+  if (counterAdult.value !== '0') {
+    btnCounterAdultMinus.classList.remove('counter-plus-minus-gray');
+    btnCounterAdultMinus.classList.add('counter-plus-minus-blue');
+  } else {
+    btnCounterAdultMinus.classList.remove('counter-plus-minus-blue');
+    btnCounterAdultMinus.classList.add('counter-plus-minus-gray');
+  };
+};
+
+btnCounterRoomsPlus.addEventListener('click', minusRooms);
+btnCounterRoomsMinus.addEventListener('click', minusRooms);
+function minusRooms() {
+  if (counterRooms.value !== '0') {
+    btnCounterRoomsMinus.classList.remove('counter-plus-minus-gray');
+    btnCounterRoomsMinus.classList.add('counter-plus-minus-blue');
+  } else {
+    btnCounterRoomsMinus.classList.remove('counter-plus-minus-blue');
+    btnCounterRoomsMinus.classList.add('counter-plus-minus-gray');
+  };
+};
+
+
+
+
+
+
+
+
+function init(id) {
+  let select;
+  let i; 
+  let option;
+  select = document.getElementById(id);
+  for (i = 1; i < 18; i ++) {
+    option = document.createElement('option');
+    option.value = option.text = i + ' years old';
+    option.classList.add('child-age-list');
+    select.add(option);
+  };
+};
+
+btnCounterChildrenPlus.addEventListener('click', () => {
+  switch (counterChildren.value) {
+    case '1':
+      init('childAgeSelect1');
+      childAgeSelect1.classList.remove('hide');
+      break;
+    case '2':
+      init('childAgeSelect2');
+      childAgeSelect2.classList.remove('hide');
+      break;
+    case '3':
+      init('childAgeSelect3');
+      childAgeSelect3.classList.remove('hide');
+      filterForm.style.width = '349px';
+      filterForm.style.height = '250px';
+    break;
+    case '4':
+      init('childAgeSelect4');
+      childAgeSelect4.classList.remove('hide');
+      filterForm.style.width = '452px';
+      filterForm.style.height = '250px';
+    break;
+    case '5':
+      init('childAgeSelect5');
+      childAgeSelect5.classList.remove('hide');
+      filterForm.style.width = '555px';
+      filterForm.style.height = '250px';
+    break;
+    case '6':
+      init('childAgeSelect6');
+      childAgeSelect6.classList.remove('hide');
+      filterForm.style.height = '291px';
+    break;
+    case '7':
+      init('childAgeSelect7');
+      childAgeSelect7.classList.remove('hide');
+      filterForm.style.height = '291px';
+    break;
+    case '8':
+      init('childAgeSelect8');
+      childAgeSelect8.classList.remove('hide');
+      filterForm.style.height = '291px';
+    break;
+    case '9':
+      init('childAgeSelect9');
+      childAgeSelect9.classList.remove('hide');
+      filterForm.style.height = '291px';
+    break;
+    case '10':
+      init('childAgeSelect10');
+      childAgeSelect10.classList.remove('hide');
+      filterForm.style.height = '291px';
+      btnCounterChildrenPlus.setAttribute('disabled', true);
+    break;
```

Comment:
very hardcode. You shouldn't specify height for everyone  children, just add and remove children due to click (not hide)

Reply:
if you can add 100 children, will you create 100 variables as well?
think about how to make it so that it does not depend on the quantity and it worked for both 10 and 100